cmd/sensor-config/README.md

```diff
@@ -0,0 +1,46 @@
+# sensor-config
+
+Build a sensor description file
+
+## output
+
+The output XML or JSON file provides a description of all sensor installations and manual collection points.
```

Comment:
suggestion(non-blocking): `The output XML (default) or JSON file provides...`

Reply:
looks good.